packages/ckeditor5-clipboard/docs/features/paste-plaintext.md

```diff
@@ -0,0 +1,45 @@
+---
+menu-title: Paste plain text
+category: features-pasting
+order: 30
+---
+
+{@snippet features/build-paste-source}
+
+# Pasting plain text
+
+The plain text pasting feature is provided through the {@link module:clipboard/clipboard~Clipboard} plugin, which is responsible for the integration with the native clipboard and intercepts all native events like `copy`, `cut` or `drop` and handles them on its side. The goal is to not allow the browser to touch the content in the rich text editor which would lead to the browser messing it up.
+
+The plain text pasting itself is further aided by the {@link module:clipboard/clipboard~PastePlainText} plugin. It detects the <kbd>Ctrl</kbd>/<kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd> keystroke during the paste and causes the pasted plain text to inherit the styles of the content it was pasted into.
+
+Pasting plain text with a double line break will turn it into a paragraph. This is a change from the previous behavior, when a single line break resulted in creating a paragraph. As of now, a single line break will be turned into a soft break upon pasting.
+
+## Demo
+
+Paste plain text between the styled paragraphs below using the <kbd>Ctrl</kbd>/<kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd> to test the style inheritance. Paste without modifier to retain styleless text. You can copy this very paragraph or one from the demo for convenience.
+
+{@snippet features/paste-plaintext}
+
+## Related features
+
+CKEditor 5 supports a wider range of paste features, including:
+* {@link features/paste-from-word Paste from Word} &ndash; Allows you to paste content from Microsoft Word and maintain the original structure and formatting.
+* {@link features/paste-from-google-docs Paste from Google Docs} &ndash; Allows you to paste content from Google Docs maintaining the original formatting and structure.
+
+## Installation
```

Comment:
Installation instructions missing, @Mgsy can you help?

Reply:
I'm not sure about this section. Actually, Paste plain text plugin is [required by Clipboard plugin](https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-clipboard/src/clipboard.js#L43) and is always enabled if the clipboard features are enabled. You can't use Paste plain text plugin without Clipboard plugin. How we should approach it?

Reply:
Maybe clipboard installation?

Reply:
I've added installation instructions for the clipboard plugin.